File: miband-sdk/src/main/java/com/khmelenko/lab/miband/listeners/NotifyListener.java
package com.khmelenko.lab.miband.listeners;

/**
 * Listener for data notifications
 *
 * @author Dmytro Khmelenko
 */
public interface NotifyListener {

    /**
     * Called when new data arrived
     *
     * @param data Binary data
     */
    void onNotify(byte[] data);
}

File: miband-sdk-kotlin/src/main/java/com/khmelenko/lab/miband/listeners/NotifyListener.kt
package com.khmelenko.lab.miband.listeners

/**
 * Listener for data notifications

 * @author Dmytro Khmelenko
 */
interface NotifyListener {

    /**
     * Called when new data arrived

     * @param data Binary data
     */
    fun onNotify(data: ByteArray)
}
